doc/versioner.md

```diff
@@ -0,0 +1,91 @@
+# Versioning a project
+
+Executing `pkgmt version` will modify the `__version__` string in the version file, and the headers in `CHANGELOG.md`
+
```

Comment:
there's not mention here of how to find the version file (should mention that you put it in a pyproject.toml or in a setup.py by default)